### PR TITLE
Update wp-accessibility.php

### PR DESCRIPTION
**Pull request by jdelia**
Adding a test for is_main_query prevents the navigation menu from vanishing on empty searches as explained by @GaryJones 

"WPA filters the query to add in a space for `s`

However, that filter also kicks in when WP does a completely un-related query to get the nav menu items (they are posts too remember). Since s=' ' & post_type= nav_menu_item results in no results returned from the query, (presumably because the search query probably limits it to certain post types, and the nav query wants it to be a different post type, it can't be A and B at the same time), then the nav menu fails to show up."

### CODE REVIEW

**Comment by joedolson** [comment]
Thanks, Jackie! I'm traveling right now, and because the GitHub version is not actually quite up to date (he says, hanging his head in shame), won't be able to merge this directly, but I'll get this taken care of as soon as I get back - thanks!

**Comment by jdelia** [comment]
thanks Joe for the reply. So glad to finally solve this one - it was driving me crazy. Thanks to @GaryJones for his help in troubleshooting it with me.

**Comment by jdelia** [comment]
Added an additional check on the string to be sure it is empty. This clears the space added to the search input field for next search after an empty one.

File: wp-accessibility.php [hunk 1]
```diff
@@ -668,7 +668,7 @@ function wpa_update_settings() {
 
 function wpa_filter( $query ) {
 	if ( ! is_admin() ) {
-		if ( isset( $_GET['s'] ) && $_GET['s'] == '' ) {
+		if ( isset( $_GET['s'] ) && ( trim($_GET['s'] ) == NULL ) && ( $query->is_main_query() ) ) {
```
Comment (GaryJones) [inline wp-accessibility.php]: If you're touching that bit, you could fix the lack of Yoda conditions too ;-)

**Comment by joedolson** [comment]
I'm closing without merging because I'm not yet really maintaining this project on GitHub, but I've added props for both of you in the readme.
